### PR TITLE
EOS-20355: Cleanup command for csm_setup

### PR DESCRIPTION
**Pull request by udayan-y21**
Signed-off-by: Udayan Yaragattikar <udayan.yaragattikar@seagate.com>

# Backend

## Problem Statement

Story Ref (if any): https://jts.seagate.com/browse/EOS-20355

## Unit testing on RPM done
<pre>
Yes
</pre>
## Problem Description
<pre>
Expose command that will cleanup all the configurations
</pre>
## Solution
https://seagate-systems.atlassian.net/wiki/spaces/~700883026/pages/347734026/CSM-setup+Cleanup+and+Reset+Scope
## Unit Test Cases
image

image

Signed-off-by: Udayan Yaragattikar <udayan.yaragattikar@seagate.com>

### CODE REVIEW

File: csm/conf/cleanup.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+from csm.conf.setup import CsmSetupError, Setup
+
+class Cleanup(Setup):
+    """
+    Delete all the CSM generated files and folders
+    """
+
+    def __init__(self):
+        super(Cleanup, self).__init__()
+        Log.info("Running Cleanup for CSM.")
```
Comment (pawankumarsrivastava) [inline csm/conf/cleanup.py:31]: Remove this log from here. We are initializing only not running anything.

File: csm/conf/cleanup.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    async def execute(self, command):
+        """
+        :param command:
+        :return:
+        """
+        try:
+            Log.info("Loading Url into conf store.")
```
Comment (pawankumarsrivastava) [inline csm/conf/cleanup.py:39]: Loading Url into conf store. ---> Loading configuration

**Review by pawankumarsrivastava — CHANGES_REQUESTED** [review]

File: csm/conf/cleanup.py (hunk 1 above)
Comment (udayan-y21) [reply csm/conf/cleanup.py:31]: Chagned to "Triggering Cleanup for CSM".. This will give identified in log file

File: csm/conf/cleanup.py (hunk 2 above)
Comment (udayan-y21) [reply csm/conf/cleanup.py:39]: Modifications done

**Review by soniyamoholkar — APPROVED** [review]

**Review by pawankumarsrivastava — APPROVED** [review]

**Pull request merged**